lib/tom_queue/version.rb

```diff
@@ -0,0 +1,3 @@
+module TomQueue
+  VERSION = "0.0.2".freeze
```

Comment:
even `1.0.0` would make sense as the removal of `stop` method can be considered as a major change here.